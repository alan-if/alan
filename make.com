$ cd compiler
$ make alan
$ cd ..
$ cd interpreter
$ make arun
