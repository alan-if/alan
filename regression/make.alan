The room  Isa location.  Is Not dark.  Description    "This is a room."    If room Is dark Then "It is dark." Else "It is not dark." End If.    Make room dark.  Exit e To room.End The room.Start At room.