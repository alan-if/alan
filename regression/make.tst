alan make
arun make < make.input
