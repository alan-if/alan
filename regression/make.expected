########## make ##########

        No warnings or errors detected.



Room
This is a room. It is not dark.

> e
Room
This is a room. It is dark.

> 

Do you want to UNDO, RESTART, RESTORE or QUIT ? 
